Cargo.toml

```diff
@@ -51,8 +51,13 @@ serde = { version = "1.0", features = ["derive"] }
 [package.metadata.config]
 mbed-crypto-version = "mbedcrypto-2.0.0"
 
+[package.metadata.docs.rs]
+features = ["docs"]
+
 [features]
-default = ["mbed-crypto-provider", "pkcs11-provider"]
+default = []
 mbed-crypto-provider = []
 pkcs11-provider = ["pkcs11", "picky-asn1-der", "picky-asn1"]
 tpm-provider = ["tss-esapi", "picky-asn1-der", "picky-asn1"]
+all-providers = ["tpm-provider", "pkcs11-provider", "mbed-crypto-provider"]
+docs = ["pkcs11-provider", "tpm-provider", "tss-esapi/docs"]
```

Comment:
Maybe worth to add a comment why `mbed-crypto-provider` is not included there!

Reply:
Dun